contributing_to_docs/doc_guidelines.adoc

```diff
@@ -473,9 +473,9 @@ xref:../../architecture/networking/routes.adoc#architecture-core-concepts-routes
 .Markup example of cross-referencing
 ----
 Rollbacks revert part of an application back to a previous deployment. Rollbacks can be performed using the REST API or
-the xref:../cli_reference/openshift_cli/get_started_cli.adoc#installing-openshift-cli[OpenShift CLI].
+the xref::../cli_reference/openshift_cli/get_started_cli.adoc#installing-openshift-cli[OpenShift CLI].
```

Comment:
Were these extra colons purposeful? One colon should be correct; I'm not sure if two would even work.

Reply:
@bergerhoffer wanted to fix build failure for topic map, but with omitted topic map doesn't really matter.